azext_iot/iothub/command_map.py

```diff
@@ -49,7 +49,7 @@ def load_iothub_commands(self, _):
         cmd_group.command("send", "iot_c2d_message_send")
         cmd_group.command("purge", "iot_c2d_message_purge")
 
-    # with self.command_group("iot hub state", command_type=iothub_state_ops, is_preview=True) as cmd_group:
-    #     cmd_group.command("export", "state_export")
-    #     cmd_group.command("import", "state_import")
-    #     cmd_group.command("migrate", "state_migrate")
+    with self.command_group("iot hub state", command_type=iothub_state_ops, is_experimental=True) as cmd_group:
```

Comment:
We may make you re-comment this out before you - keep this comment open for now.

Reply:
This is in a feature branch now (dev reverted), so it is OK to keep it uncommented.